server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -73,6 +73,7 @@
         public static final String FORCE_MERGE = "force_merge";
         public static final String FETCH_SHARD_STARTED = "fetch_shard_started";
         public static final String FETCH_SHARD_STORE = "fetch_shard_store";
+        public static final String LOGICAL_REPLICATION = "logical_replication";
```

Comment:
Do we need to document this one for thread pool settings or extend JMX ?

Reply:
👍 I've added it to the mbean and to the documentation.